### DISCUSSION TITLE
Retrieving table data

### DISCUSSION THREAD

**Discussion by Gurshaan7** [aptos-labs/aptos-developer-discussions#169] [Questions]
### Describe your question in detail.

I was using API to read the table value corresponding to a key , I didn't get any error in the execution but the output I receive is in gibberish form , Is that output is in some form of encoded data that I need to decode?

### What have you tried or looked at?  Or how can we reproduce the error?

Just use API to fetch any table value corresponding to a key.

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

REST API

**Comment by BriungRi** [comment]
Can you share more about the `Table` type and which API you are calling? Ideally, you can share enough for us to replicate your problem on our end.

**Comment by Gurshaan7** [comment]
Basically I want to fetch the values of bin field in  Pool resource of module Pool deployed on this address ->0x54cb0bb2c18564b86e34539b9f89cfe1186e39d89fce54e1cd007b8e61673a85 and the Pool resource is on this address -> 0xa0d8702b7c696d989675cd2f894f44e79361531cff115c0063390922f5463883.
Request is working but the value I am getting is in Gibberish form .

My Code:
`const axios = require('axios');

const options = {
  method: 'POST',
  url: 'https://api.mainnet.aptoslabs.com/v1/tables/0xc50d41151fbe707675396fd1ee3a95474139aa69a42d85c162888ac180dc725/item',
  headers: {
    'Content-Type': 'application/json',
    Accept: 'application/json, application/x-bcs'
  },
  data: {
    key: '8388608',
    key_type: 'u32',
    value_type: '0x54cb0bb2c18564b86e34539b9f89cfe1186e39d89fce54e1cd007b8e61673a85::pool::Bin',

  }
};

try {
  axios.request(options).then((response) => {
    console.log(response);
  });
} catch (error) {
  console.error(error);
}`
